app/controllers/application_controller.rb

```diff
@@ -5,6 +5,155 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "fwitter_secret"
   end
 
+  get '/' do
+    #welcome to fwitter!!!
+    erb :index
+  end
+
+  get '/tweets' do
+    #index of all tweets
+    if session[:user_id]
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    #load form to create a new tweet
+    if session[:user_id]
+      erb :'tweets/create'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    #creates new tweet
+    if params[:content] == ""
+      redirect to "/tweets/new"
```

Comment:
I think you could use [ActiveRecord validations](https://learn.co/tracks/full-stack-web-development/sinatra/activerecord/user-authentication-in-sinatra) (e.g. `validates_presence_of :content`) to perform these types of checks more easily.